Buy screen drink customizations

lmillet3804 opened the pull request:
Redid Drink Customizations:

- used URL params to get page titles from DB and render cooresponding customizations
- created formatOrderFull method to render order nicely (including drink customizations)
- redux changes to add drink customizations instead of setting them directly (create object)

lmillet3804 committed:
first attempt at trying to fix drink customizations problem

-no way to save in redux
-hardcoded milkChoice and sweetenerChoice

lmillet3804 committed:
committing to change branches

lmillet3804 committed:
Merge branch 'main' of https://github.com/meal-match/frontend into buy-screen-drink-customizations

lmillet3804 committed:
Redid Drink Customizations:

- used URL params to get page titles from DB and render cooresponding customizations
- created formatOrderFull method to render order nicely (including drink customizations)
- redux changes to add drink customizations instead of setting them directly (create object)

lmillet3804 committed:
Merge branch 'main' of https://github.com/meal-match/frontend into buy-screen-drink-customizations

TheGoldenPatrik1 commented on app/buy/milkChoice.js:
```diff
@@ -0,0 +1,43 @@
+
+    const moveForward = (sweetener) => {
+        // do something with sweetener
+        const something = { sweetener, something } // just to make eslint happy
```
Remove this line of code. You should be able to add an eslint-ignore comment above line 16 that will fix the eslint issue. Just view the problem; it should be one of the quick fixes.

TheGoldenPatrik1 commented on app/buy/milkChoice.js:
```diff
@@ -0,0 +1,43 @@
+                    { name: 'Splenda' },
+                    { name: 'Equal' },
+                    { name: 'Sweet N Low' },
+                    { name: 'No Sweetener' }
```
Wait aren't we supposed to be getting these values from the db?

TheGoldenPatrik1 commented on app/buy/sweetenerChoice.js:
```diff
@@ -0,0 +1,30 @@
+const sweetenerChoice = () => {
+    const moveForward = (sweetener) => {
+        // do something with sweetener
+        const something = { sweetener, something } // just to make eslint happy
```
Remove, per earlier comment.

TheGoldenPatrik1 commented on app/buy/sweetenerChoice.js:
```diff
@@ -0,0 +1,30 @@
+                    { name: 'Splenda' },
+                    { name: 'Equal' },
+                    { name: 'Sweet N Low' },
+                    { name: 'No Sweetener' }
```
Update, per earlier comment.

TheGoldenPatrik1 commented on store/actions/orderActions.js:
```diff
@@ -126,6 +127,28 @@ export const setDrink = (drink) => (dispatch, getState) => {
+        if (order.orderLoading) {
+            return
+        }
+        try {
```
This is just a simple redux dispatch. Do we really need to try/catch? I'm not even sure we need to check for orderLoading. Just make it consistent with other, similar redux events.

TheGoldenPatrik1 commented on utils/index.js:
```diff
@@ -3,3 +3,4 @@ export * from './notifications'
 export * from './strings'
 export * from './time'
 export * from './error'
+export * from './order'
```
This is nit-picky but can we make this list of exports alphabetical?

lmillet3804 committed:
Addressed PR Comments:

- removed old files added from previous commit
- simplified redux addDrinkCustomization option
- alphabetized exports
